.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -32,7 +32,7 @@ body:
     id: helix-log
     attributes:
       label: Helix log
-      description: See `hx -h` for log file path
+      description: See `hx -h` for log file path, if you can reproduce the issue run `RUST_BACKTRACE=1 helix -v` to generate a more detailed log file.
```

Comment:
```suggestion
      description: See `hx -h` for log file path. If you can reproduce the issue run `RUST_BACKTRACE=1 hx -v` to generate a more detailed log file.
```

Reply:
Also, I'd err on the side of more verbosity with `-vvvv`